server.go

```diff
@@ -1,26 +1,34 @@
 package main
 
 import (
-	"gowebapp/routers"
+	"flag"
+	"strings"
+
+	datastructure "github.com/IBM/go-web-app/datastructure"
+	routers "github.com/IBM/go-web-app/routers"
+	stringutils "github.com/alessiosavi/GoGPUtils/string"
```

Comment:
We also cant import anything from your personal github especially to run just the one method